esp/src/eclwatch/ESPResult.js

```diff
@@ -77,6 +77,9 @@ define([
             }
         },
         preProcessResponse: function (response, request) {
+            if (response.Total == -1 || response.Total === 9223372036854776000 || response.Total === Number.MAX_VALUE) {
```

Comment:
Are we assure that the number is the case all the time? Since you are using the bug report's value as your test.